File: API/FilesUpoadAPI/FilesUpload.Domain/FileToUpload.cs
using FilesUpload.Domain.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FilesUpload.Domain
{
    public class FileToUpload : BaseDomainClass
    {
        public string FileType { get; set; }
        public string FileName { get; set; }
        public string Path { get; set; }
        public int FileSize { get; set; }
    }
}

File: Client/FilesUplaodClient/src/app/File-Upload/Models/FileToUpload.ts
export interface FileToUploadDto {
  id: number;
  fileName: string;
  fileSize: number;
  fileType: string;
  uploadDate: Date;
}

export interface FileToUpload {
  FileType: string;
  FileName: string;
  FileDBName: string;
  Path: string;
  FileSize: number;
}
